src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -733,27 +734,27 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
      *  runtime reflection universe, and if yes and if we've not yet loaded the requested info, then to commence initialization.
      */
     final def getFlag(mask: Long): Long = {
-      if (!isCompilerUniverse && !isThreadsafe(purpose = FlagOps(mask))) initialize
-      flags & mask
+      var f = if ((mask & PhaseIndependentFlags) == mask) rawflags else flags
```

Comment:
👍 for this part, I tried something [similar in a WIP branch](https://github.com/scala/scala/compare/2.13.x...retronym:faster/round3#diff-6cc67f90cb479803e84e8068d49663fbR816) by special casing `isError`, but this looks cleaner.